schemas/pe.yml

```diff
@@ -0,0 +1,50 @@
+---
+- name: pe
+  title: PE Header
```

Comment:
Please write it in full here. Perhaps "Portable Executable (PE) Header"

Reply:
what's the impact on the sort order? I remember running into this problem in another place

Reply:
The reason I was asking this isn't for the sorting, but more for clarity. But looking again in both listing in the docs (see page body and sidebar [here](http://ecs_731.docs-preview.app.elstc.co/guide/en/ecs/master/ecs-field-reference.html)), I think I'm good with keeping the title like it is now, and having it spelled out in the description.

So never mind this request.

To answer your question, the sorting is done on the short name (here "pe"). This is done implicitly by Python dicts, since it's used as the key name, in the in memory and intermediate YAML files. We could adjust the asciidoc generator to sort by full name instead, though.